run/archiveloop

```diff
@@ -385,10 +385,14 @@ function set_time () {
     sleep 2
   done
   log "Time still not set, attempting to force it"
+  log "Stopping ntp process ..."
+    /etc/init.d/ntp stop
```

Comment:
This should be checking the return code, so that archiveloop doesn't abort if there's a failure.
So either:
```
if ! /etc/init.d/ntp stop
then
  log "failure stopping ntp"
fi
```
or at the very least:
```
/etc/init.d/ntp stop || true
```

Reply:
Thanks for direction. Traveling next few days and will review when I return over the weekend.